Update championships (#2103)
tpwres/teczki-polskiego-wrestlingu: content/c/ptw-underground-championship.md

## History

The title was first mentioned in a December 2024 interview for the [MyWrestling YouTube channel][pawłowski-my-wrestling-live]. Pawłowski stated that a tournament would be held to crown the inaugural champion. Unlike the [PTW Women's Championship](@/c/ptw-womens-championship.md) title tournament, this one would happen entirely at [one show](@/e/ptw/2025-02-15-ptw-wrestlingowe-walentynki.md). 

The initial champion was [Max Speed](@/w/max-speed.md), who won it in a Ladder Match against [Vincent Caravaggio](@/w/vincent-caravaggio.md). His reign was however cut short due to his unexpeted firing from PTW. Following his departure, the #1 Contender who won [Gold Rush Rumble](@/e/ptw/2025-06-28-ptw-zloto-dla-zuchwalych.md) - [Nano Lopez](@/w/nano-lopez.md) - faced [Daniel Razor](@/w/daniel-razor.md) for the vacated championship, with the latter winning it to become the second champion.

## Belt design

> The championship belt is rather bulky in its shape and size. Main plate's background is made with metal in grey colour with a huge orange elipse with Prime Time Wrestling logo in the centre. Above and beneath this elipse there is "UNDERGROUND" and "CHAMPION" written in red, which were deliberately obliterated to look a bit worn. Side plates are rather simple, and they depict a clenched black fist on a orange circular blackground.

Reviewer: ```suggestion
The championship belt is rather bulky in its shape and size. The main plate's background is made of grey-coloured metal with a huge orange ellipse, bearing the Prime Time Wrestling logo in the centre. Above and beneath this ellipse "UNDERGROUND" and "CHAMPION" are written in red, deliberately obliterated to look a bit worn. The side plates are rather simple, depicting a clenched black fist on an orange circular background.
```
